pkg/arvo/mar/metadata/action.hoon

```diff
@@ -0,0 +1,7 @@
+/-  *metadata-store
+|_  act=metadata-action
+++  grab
```

Comment:
Maybe also write the trivial `+noun:grow`, for completeness' sake.

Reply:
Matilde’s PR includes a grow arm to noun and json already. I’ll merge this when I get back from workout at noon

Reply:
Alternatively, feel free to merge sooner. The conflict in that file is trivial